### PR TITLE
add check for https when downloading mathjax

### PR DESCRIPTION
**Pull request by gravitypersists**

### CODE REVIEW

File: config.coffee [hunk 1]
```diff
@@ -42,7 +42,8 @@ define [], ->
     for k,v of config.paths
       config.paths[k] = "#{path}/lib/#{v}"
 
-    config.paths['cdn.mathjax'] = "http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=AM_HTMLorMML-full.js"
+    h = if window.location.href.match(/^https\:/) then "https" else "http"
```
Comment (rjz) [inline config.coffee]: [`window.location.protocol`](https://developer.mozilla.org/en-US/docs/Web/API/window.location)

```
h = window.location.protocol
config.paths['cdn.mathjax'] = h + '//cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=AM_HTMLorMML-full.js'
```

**Comment by dylanpyle** [comment]
What's wrong with just requesting `//cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=AM_HTMLorMML-full.js` ? See http://tools.ietf.org/html/rfc3986#section-4.2

**Comment by dylanpyle** [comment]
Hmm, for that matter the API could probably just return URLs in that format with minimum damage...

**Comment by rjz** [comment]
Good Q...kind of missed an obvious one, here, didn't we?

**Pull request merged**